### PR TITLE
EVG-16351 Fix timing for splunk logging

### PR DESCRIPTION
**Pull request by khelif96**
[EVG-16351](https://jira.mongodb.org/browse/EVG-16351)

### Description 
I think this timing calculation was wrong and was logging time.Duration instead of a millisecond value.

### CODE REVIEW

**Review by malikchaya2 — APPROVED** [review]

File: graphql/util.go [hunk 1]
```diff
@@ -532,16 +532,16 @@ func generateBuildVariants(sc data.Connector, versionId string, searchVariants [
...
-		"timeToBuildTasks":  timeToBuildTasks,
-		"timeToGroupTasks":  timeToGroupTasks,
-		"timeToSortTasks":   timeToSortTasks,
-		"totalTime":         totalTime,
-		"versionId":         versionId,
-		"taskCount":         len(tasks),
-		"buildVariantCount": len(result),
+	grip.InfoWhen(time.Duration(totalTime.Milliseconds()) > time.Second*2, message.Fields{
+		"Ticket":             "EVG-14828",
+		"timeToFindTasksMS":  timeToFindTasks.Milliseconds(),
```
Comment (malikchaya2) [inline graphql/util.go:537]: Wont this still be time.duration just converted to Milliseconds?

**Review by malikchaya2 — CHANGES_REQUESTED** [review]

File: graphql/util.go (hunk 1 above)
Comment (malikchaya2) [reply graphql/util.go:537]: (instead of nanoseconds)

Comment (malikchaya2) [reply graphql/util.go:537]: https://go.dev/play/p/F9dtew60dob

Comment (khelif96) [reply graphql/util.go:537]: I think we use milliseconds in some other splunk logging it's also arguably a more readable value.

Comment (khelif96) [reply graphql/util.go:537]: image

**Review by malikchaya2 — APPROVED** [review]

**Pull request merged**